### ISSUE TITLE
Variational Autoencoder or Autoencoder?

### ISSUE THREAD

**Issue by Xuanhao-Luo** [kakaobrain/rq-vae-transformer#30]
Thank you for releasing the excellent RQ-VAE codebase. After reading both the paper and implementation, I noticed that the encoder outputs deterministic latent features and the training loss does not include a KL divergence term. Could you please clarify whether RQ-VAE should be interpreted as a Variational Autoencoder or as a Residual-Quantized Autoencoder (a deterministic model)?

Thank you so much for your time and clarification!
